browser_history/cli.py

```diff
@@ -92,9 +92,11 @@ def main():
         "bookmarks": {"var": b_outputs, "fun": get_bookmarks},
     }
 
-    assert (
-        args.type in fetch_map.keys()
-    ), f"Type should be one of all, {AVAILABLE_TYPES}"
+    if args.type not in fetch_map:
+        utils.logger.error(
+            "Type %s is unavailable." "Check --help for available types", args.type
```

Comment:
I believe there should be a space between these two sentences

Reply:
Even the invalid browser message needed this edit

Reply:
It is done.